### DISCUSSION TITLE
Best setting for continuous charge

### DISCUSSION THREAD

**Discussion by genik76** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#927] [General]
I have a 16" M1 MBPro and i mostly have it on charger. What AlDente setting is the best for the batteries longevity?

**Comment by bugac** [comment]
> I have a 16" M1 MBPro and i mostly have it on charger. What AlDente setting is the best for the batteries longevity?

IMHO 70-80%

**Comment by genik76** [comment]
Is there a video tutorial on how to set AlDente?

**Reply by bugac** [reply]
> Is there a video tutorial on how to set AlDente?

What you want to setup? Set charge level you want and leave it to work

**Reply by genik76** [reply]
I just need to set the charging level to 80% and thats it? I can have it constantly on the charger?

**Reply by bugac** [reply]
> I just need to set the charging level to 80% and thats it? I can have it constantly on the charger?

yes, it will not charge above 80%

**Reply by genik76** [reply]
If i close the laptop's lid will AlDente's settings still work?

**Reply by bugac** [reply]
> If i close the laptop's lid will AlDente's settings still work?

https://apphousekitchen.com/faq/
also read every feature explanation here https://apphousekitchen.com/category/featureexplanation/

**Comment by genik76** [comment]
When having it on continuous charge at 80%, is it good for the battery, every now and then to stop AlDente and discharge to 20-30%?
